src/main/java/wolox/training/models/Book.java

```diff
@@ -117,13 +124,15 @@ public int getPages() {
 
     public void setPages(int pages) {
         this.pages = pages;
+        Preconditions.checkNotNull(pages, "Illegal Argument, pages is Null.");
```

Comment:
same here, negative pages shouldn't be allowed

Reply:
Fixed at https://github.com/wolox-training/yh-java/pull/11/commits/6d7dd31f8f886894e790e10b107b0968347e49a7 commit